src/grid/PartRepresentations.php

```diff
@@ -60,10 +61,17 @@ protected function fillRepresentations()
                     'selling_time',
                 ]
             ] : null,
-            'profit-report' => Yii::$app->user->can('order.read-profits') ? [
+            'profit-report' => $user->can('order.read-profits') ? [
                 'label' => Yii::t('hipanel', 'profit report'),
                 'columns' => ProfitColumns::getColumnNames(['checkbox', 'buyer', 'company_id', 'serial', 'partno']),
             ] : null,
+            'admin' => $user->can('admin') && $user->can('order.create') ? [
+                'label' => 'admin',
```

Comment:
перевод не хо добавить?

```suggestion
                'label' => Yii::t('hipanel:stock', 'Administrative'),
```

```php
'Administrative' => 'Административный',
```